packages/salesforcedx-vscode-apex/src/codecoverage/colorizer.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { CodeCoverageResult } from '@salesforce/apex-node';
+import { Global } from '@salesforce/core';
```

Comment:
Dumb q: should we create a place in utils-vscode for getting all the things from core like this? Maybe a `PathService` that will give us path locations similar to how we're using ConfigUtils today?   Has utility functions that looks something like `PathService.getApexPath();`
That way we can limit core to just the utils packages

Reply:
💯 yes please

Reply:
Yep!  Will wait for PathService to be introduced in @RitamAgrawal's PR here: https://github.com/forcedotcom/salesforcedx-vscode/pull/4408/files and once that is merged, will add functions for them there.